src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -46,6 +47,20 @@ public interface IDynamoViewModel : INotifyPropertyChanged
         ObservableCollection<WorkspaceViewModel> Workspaces { get; set; } 
     }
 
+    public class SimpleViewModel<T> : ViewModelBase, INotifyPropertyChanged
```

Comment:
experiment?

Reply:
apparently you can use old style tuple items as bindings - you cannot use new style value tuples though as those members are fields - anyhow, something I consider I guess.

Reply:
https://stackoverflow.com/questions/43208852/is-it-possible-to-bind-to-a-valuetuple-field-in-wpf-with-c7

Reply:
Maybe not the right place for it, but I thought this could be a good way to to it. I'm not a fan of tuples, specially because the property names are positional and have no meaning.